src/gui/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ find_package(Qt5 COMPONENTS Widgets Qml LinguistTools REQUIRED)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+configure_file(version.h.in version.h)
```

Comment:
I think version.h is a part of core library not gui. So please transfer version.h to core library and I don't see version changes in main.cpp.

Reply:
fixed